KeepingYouAwake/KYAAppController.m

```diff
@@ -242,18 +254,25 @@ - (void)terminateTimer
 
 #pragma mark - Menu Delegate
 
-- (IBAction)selectTimeInterval:(id)sender
+- (IBAction)selectTimeInterval:(NSMenuItem *)sender
 {
     if([self.sleepWakeTimer isScheduled])
     {
         [self.sleepWakeTimer invalidate];
     }
-    
-    __weak typeof(self) weakSelf = self;
-    dispatch_async(dispatch_get_main_queue(), ^{
-        NSTimeInterval seconds = (NSTimeInterval)[sender tag];
-        [weakSelf activateTimerWithTimeInterval:seconds];
-    });
+	
+	if (sender.alternate)
+	{
+		[self setDefaultTimeInterval: sender.tag];
+	} else {
```

Comment:
Can you please move `else` and `{` to separate lines?

*(I know, I have a line-wasting codestyle :wink:)*